client/a8c-for-agencies/sections/marketplace/wpcom-overview/index.tsx

```diff
@@ -77,6 +84,10 @@ function WpcomOverview() {
 		}
 	}, [ creatorPlan, isLicenseCountsReady, licenseCounts?.products ] );
 
+	// For referral mode we only display 1 option.
+	const dispalyQuantity = referralMode ? 1 : ( selectedTier.value as number ) - ownedPlans;
```

Comment:
There is a typo here. 
```suggestion
	const displayQuantity = referralMode ? 1 : ( selectedTier.value as number ) - ownedPlans;
```

Reply:
Thanks for spotting this! Nice catch.